### DISCUSSION TITLE
1Password ssh key support

### DISCUSSION THREAD

**Discussion by sweh** [FiloSottile/age#563] [General]
Hi there,

am I missing something from the manual or are ssh keys (not passphrase) stored in 1Password [1] not supported? Are there any plans to support this?

As 1Password provides a custom ssh agent, the question in general might be: Are custom ssh agents supported and how?

Thanks and cheers,
  Sebastian

[1] https://developer.1password.com/docs/ssh/

**Comment by WillPower3309** [comment]
This is discussed in https://github.com/FiloSottile/age/discussions/244
